src/Module.zig

```diff
@@ -608,18 +608,23 @@ pub const Decl = struct {
     }
 
     pub fn srcToken(decl: Decl) Ast.TokenIndex {
-        const tree = &decl.namespace.file_scope.tree;
+        const tree = &decl.getFileScope().tree;
         return tree.firstToken(decl.src_node);
     }
 
     pub fn srcByteOffset(decl: Decl) u32 {
-        const tree = &decl.namespace.file_scope.tree;
+        const tree = &decl.getFileScope().tree;
         return tree.tokens.items(.start)[decl.srcToken()];
     }
 
     pub fn renderFullyQualifiedName(decl: Decl, writer: anytype) !void {
         const unqualified_name = mem.spanZ(decl.name);
-        return decl.namespace.renderFullyQualifiedName(unqualified_name, writer);
+        return decl.src_namespace.renderFullyQualifiedName(unqualified_name, writer);
+    }
+
+    pub fn renderFullyQualifiedDebugName(decl: Decl, writer: anytype) !void {
```

Comment:
I was getting confused by type names like `mem.Allocator.mem.Allocator`, so this alternate method renders them as `mem/Allocator.zig:mem.Allocator`. Which now that I think about it might indicate another namespace bug... but the format conveys more information regardless. This is what the debug report uses now.